File: src/main/java/rpa/model/EventResponseDTO.java
package rpa.model;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
public class EventResponseDTO {
    private LocalDateTime startTime;
    private String type;
    private String leagueName;
    private MatchResponseDTO match;

}

File: src/main/kotlin/rpa/model/EventResponseDTO.kt
package rpa.model

import lombok.Builder
import org.joda.time.DateTime
import java.time.LocalDateTime
import java.util.*

data class ResponseDTO(val events : List<EventResponseDTO>)

data class EventResponseDTO (
        var startTime : LocalDateTime,
        var type : String,
        var leagueName : String,
        var match : MatchResponseDTO
)

data class MatchResponseDTO(
        var matchId : String,
        var gameCount : Int,
        var teams : List<TeamsResponseDTO>
)

data class TeamsResponseDTO(
        var code : String,
        var name : String,
        var image : String,
        var recordWins : Int,
        var recordLosses : Int
)